examples/node/README.md

```diff
@@ -55,6 +45,5 @@ TUTORIAL
 You can find a more detailed tutorial in [gRPC Basics: Node.js][]
 
 [homebrew]:http://brew.sh
-[linuxbrew]:https://github.com/Homebrew/linuxbrew#installation
-[gRPC install script]:https://raw.githubusercontent.com/grpc/homebrew-grpc/master/scripts/install
-[gRPC Basics: Node.js]:https://github.com/grpc/grpc/blob/master/examples/node/route_guide/README.md
+[Install gRPC Core]:../../INSTALL
```

Comment:
That's not really the point.  
It does not make sense to have [these instructions] (https://github.com/grpc/grpc/tree/master/src/node#installation) for installing gRPC Node, yet link to something else here.  We've had some feedback about this (inconsistent installation directions) and are aiming to fix it for beta.